policy-controller/k8s/index/src/pod.rs

```diff
@@ -46,18 +46,18 @@ pub(crate) type PortMap<V> =
 pub(crate) struct PortHasher(u16);
 
 /// Gets the set of named TCP ports from a pod spec.
-pub(crate) fn tcp_port_names(spec: Option<k8s::PodSpec>) -> HashMap<String, PortSet> {
+pub(crate) fn tcp_port_names(spec: &Option<k8s::PodSpec>) -> HashMap<String, PortSet> {
```

Comment:
The naming around TCP is a bit confusing here because in this function we mean TCP as opposed to UDP.  But then we use these "tcp" port names when looking at HTTP probes (and ignoring TCP probes).  Anyway, this is all correct, but TCP is used to mean two different things: TCP vs UDP and TCP vs HTTP.

Reply:
I renamed to `port_names` and clarified in the doc comment that this gets non-UDP port names.